test.py

```diff
@@ -0,0 +1,13 @@
+from atlassian import Jira
+
+j = Jira(
+    url='https://zazucli.atlassian.net/',
+    username='nhwiles@gmail.com',
+    password='ZdgU1Xtft8OoQu1ZFNu085D6')
+
+me = j.issue('ZZ-5', fields='assignee')
+print(me['key'])
+try:
+    j.assign_issue(me['key'], 'asdf')
+except Exception as e:
+    print(e)
```

Comment:
not sure if `tox` calls `pytest` with the `-s` command but it might be necessary to see any print statements like this.

Reply:
This file was not meant to be part of this change :)